dev-tools/mage/common.go

```diff
@@ -353,6 +353,69 @@ func unzip(sourceFile, destinationDir string) error {
 	return nil
 }
 
+// Tar compress a directory using tar + gzip algorithms
+func Tar(src string, targetFile string) error {
+	fmt.Printf(">> creating TAR file from directory: %s, target: %s\n", src, targetFile)
+
+	f, err := os.Create(targetFile)
+	if err != nil {
+		return fmt.Errorf("error creating tar file: %w", err)
+	}
+	defer f.Close()
+
+	// tar > gzip > buf
```

Comment:
Nit. :slightly_smiling_face: 
```suggestion
	// tar > gzip > file
```